lib/RedditScraper.ts

```diff
@@ -70,10 +70,13 @@ export class RedditScraper {
 			},
 		};
 
+		requestOptions.qs = {};
+		
+		if (options.Records) {
+			requestOptions.qs["limit"] = options.Records;
```

Comment:
It seems object access via string literals is disallowed in TSLint - would it be possible to update this so that the code is inline with the linting as recommended below?

```suggestion
			requestOptions.qs.limit = options.Records;
```